tech/express.ts

```diff
@@ -1,16 +1,19 @@
-const express = require('express');
-const cors = require('cors')
-const app = express();
+import * as express from 'express';
 const reload = require('reload');
 const watch = require('node-watch');
```

Comment:
Why not use `import * as ... from '...';`?

Reply:
I'm waiting https://github.com/yuanchuan/node-watch/pull/87 and some dependencies have no types, I'll check if I can contribute to add types or if I should use another dependency for the others.

Reply:
Ok for this import, I know there is other require in code to move.